about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>confrince</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <nav class="contact-nav">
+      <ul>
+        <li>
+          <a href="#"
+            ><img src="./imges/icons/facebook-16.ico" alt="facebook"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img src="./imges/icons/twitter-16.ico" alt="twitter"
+          /></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </nav>
+    <section class="cover">
+      <nav class="header">
+        <img
+          src="./imges/icons/dropdown-menu.svg"
+          alt="dropdown-menu"
+          class="dropdown-menu display-non-mobile"
+        />
+        <div class="display-non-desktop disktop-nav">
+          <div class="close-btn">&#x2715;</div>
+          <a href="./index.html">
+            <img
+              src="./imges/icons/main-logo.svg"
+              alt="header-icon"
+              class="logo"
+            />
+          </a>
+          <ul>
+            <li>
+              <a href="about.html"> <span>About</span></a>
+            </li>
+            <li><a href="./index.html">program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">News</a></li>
+            <li><h4 class="box">camping</h4></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="cover-about">
+        <h1>
+          CREATIVE COMMONS
+          <br />
+          GLOBAL SUMMIT 2021
+        </h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet
+          recusandae quia et? Ut cumque ullam magni Lorem ipsum dolor sit amet
+          consectetur adipisicing elit. Amet recusandae quia et? Ut cumque ullam
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet
+          recusandae quia et? Ut cumque ullam magni Lorem ipsum dolor sit amet
+          consectetur adipisicing elit. Amet recusandae quia et? Ut cumque ullam
+        </p>
+        <h3>2021.10.15</h3>
+        <h6>@ Lorem ipsum dolor sit amet consectetur</h6>
+      </div>
+    </section>
```

Comment:
- [ ] Adjust the font size and font weight of the `h1` tag
- [ ] The `p` tag should have a white background
- [ ] Kindly fix the overlapping section
- [ ] Reduce the size (height) of the image as mentioned by the previous reviewer.

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124241129-62992f80-db13-11eb-962f-fbda8a3efc1e.png) | ![image](https://user-images.githubusercontent.com/16058470/124241325-91afa100-db13-11eb-8503-0dc91f9ddbaf.png)
-- | --
![image](https://user-images.githubusercontent.com/16058470/124241519-b9066e00-db13-11eb-9ad8-9bbad052af95.png) | ![image](https://user-images.githubusercontent.com/16058470/124241694-e4895880-db13-11eb-8cc3-cf7886a448c8.png)

Reply:
- [ ] I need to reinforce this request, you need to put more attention in the layout, all this website works with rectangles (except the speaker's section), your Images form "See the past" must be rectangles, not squares, and double-check the size of the width on your paragraphs. Plus the color of the "See the past" title is wrong. Put attention to the capital letter too.

Your design:

![image](https://user-images.githubusercontent.com/60631456/124290572-3385bc80-db19-11eb-8645-106d6a69f457.png)

Original design:

![image](https://user-images.githubusercontent.com/60631456/124290695-5dd77a00-db19-11eb-8bfa-54b319985e33.png)

- [ ] Reduce the size of the logo and add a border to mimic the rectangle, check out the difference:

Your design:

![image](https://user-images.githubusercontent.com/60631456/124290890-99724400-db19-11eb-8da1-bc802794317a.png)

Original design;

![image](https://user-images.githubusercontent.com/60631456/124291024-bc045d00-db19-11eb-8b34-64460dae460f.png)

- [ ] Add the little orange underline, you missed it